assets/js/main.js

```diff
@@ -0,0 +1,323 @@
+/////////////////////////////
+// Declares DOM elements
+/////////////////////////////
+
+var listDrawPile = document.querySelector('.DrawPile')
+var listHumanPlayerPile = document.querySelector('.HumanPlayerPile')
+var listComputerPlayerPile = document.querySelector('.ComputerPlayerPile')
+var listDiscardPile = document.querySelector('.DiscardPile')
+var startButton = document.querySelector('.StartGame')
+var resetButton = document.querySelector('.ResetGame')
+var drawCardButton = document.querySelector('.DrawCard')
+var HumanPlayerCardsArr = [] // Will be defined at startGame()
+var GameMessage = document.querySelector('.GameMessage')
+
+/////////////////////////////
+// Functions for game action events
+/////////////////////////////
+
+function startGame () {
+  if (GameStarted === false) {
+    DrawPile = shuffle(DrawPile)
+
+    // while ((DrawPile[DrawPile.length-1].value === vWild) || (DrawPile[DrawPile.length-1].value === DrawFour)) {
+    //   //ensures no Wild or Wild DrawFour for starting card in DiscardPile
+    //   DrawPile = shuffle(DrawPile)
+    // }
+
+    DiscardPile = dealCard(1)
+    HumanPlayerPile = dealCard(7)
+    ComputerPlayerPile = dealCard(7)
+
+
+    // Presents Human Player's Pile
+    HumanPlayerPile.forEach(function (card, index) {
+          var listCard = document.createElement('li')
+          listCard.innerHTML = '<a id="hp'+index+'" href="#">'+card.label+'</a>'  //let listed card be clickable

```

Comment:
shouldn't use inner html to create card. Take a look at https://jeremiahalex.gitbooks.io/wdi-sg/content/02-js-jquery/js-dom-manipulation/ under `The Case Against .innerHTML`